src/api/lib/tasks/dev.rake

```diff
@@ -46,10 +46,24 @@ namespace :dev do
   task :bootstrap, [:old_test_suite] => %i[prepare environment] do |_t, args|
     args.with_defaults(old_test_suite: false)
 
-    begin
-      Rake::Task['db:version'].invoke
-    rescue StandardError
-      puts 'Creating and seeding the database...'
+    database_exists = false
+
+    puts 'Checking if database exists...'
+    if RailsVersion.is_7_2?
+      # Since Rails 7.2 `Rake::Task['db:version'].invoke` does not raise exception anymore if the database does not exist.
+      # So we need to check if the database exists before running the task.
+      database_exists = true if ActiveRecord::Base.connection.database_exists?
+    else
+      begin
+        Rake::Task['db:version'].invoke
+        database_exists = true
```

Comment:
This will not happen if the line above raises...

Reply:
That's exactly the expected behavior.
This is how we were doing before: if it raises, it means the database does not exist. Then, we create it.

Reply:
Ah so you want to do express this?

```
begin
  Rake...
rescue StandardError
  database_exists = false
end
```

Reply:
Yeah. I start with the statement `database_exists = false` so I did the reverse logic